Rivet/Functions/Checkpoint-Migration.ps1

```diff
@@ -0,0 +1,66 @@
+
+function Checkpoint-Migration
+{
+    <#
+    .SYNOPSIS
+    Checkpoints the current state of the database so that it can be re-created.
+
+    .DESCRIPTION
+    The `Checkpoint-Migration` function captures the state of a database after all migrations have been applied. The
+    captured state is exported to a `schema.ps1` file that can be applied with Rivet to re-create that state of the
+    database. Before capturing the current state, a Rivet Push is invoked to ensure all migrations have been applied.
+
+    .EXAMPLE
+    Checkpoint-Migration -Database $Database -Environment $Environment -ConfigFilePath $ConfigFilePath
+
+    Demonstrates how to checkpoint a migration.
+    #>
+    [CmdletBinding()]
+    param(
+        # The database(s) to migrate.
+        [Parameter(Mandatory)]
+        [String[]] $Database,
+
+        # The environment you're working in.
+        [String] $Environment,
+
+        # The path to the Rivet configuration file. Default behavior is to look in the current directory for a `rivet.json` file.
+        [String] $ConfigFilePath,
+
+        # The output path for the schema.ps1 file. If not provided the path will default to the build root.

```

Comment:
Update the parameter doc to indicate that the default path to the `schema.ps1` file is the same directory as the `rivet.json` file.